content/docs/setup/kubernetes/quick-start/index.md

```diff
@@ -12,13 +12,14 @@ To install and configure Istio in a Kubernetes cluster, follow these instruction
 1. [Download the Istio release](/docs/setup/kubernetes/download-release/).
 
 1. [Kubernetes platform setup](/docs/setup/kubernetes/platform-setup/):
-  * [Minikube](/docs/setup/kubernetes/platform-setup/minikube/)
+  * [Alibaba Cloud](/docs/setup/kubernetes/platform-setup/alicloud/)
```

Comment:
not sure how this got un-alphabetized.  We were very clear that project/product lists should be alphabetized in docs.  Thanks for fixing here.

Reply:
No problem.  It looks like 'release-1.0' has everything just fine (once I've fixed OCI), so that was on me. Both Docker and OCI are out of order in 'master', I'll go fix that.